### PR TITLE
fix(lib): ivy compatibility

### LINKED ISSUE #11: doesn't work with next.11
```
ERROR in /Users/fxck/www/zeropsmonorepo/node_modules/@typebytes/ngx-template-streams/fesm5/typebytes-ngx-template-streams.js 114:42-59
"export 'ɵNG_COMPONENT_DEF' was not found in '@angular/core'
```

### PR DESCRIPTION
**Pull request by fxck**
NG_COMPONENT_DEF -> NG_COMP_DEF, fixes https://github.com/typebytes/ngx-template-streams/issues/11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ We like reactive programming and with this experiment we hope to push this forwa
 
 ## Features
 
-- ✅ Works with both `ViewEngine` and `Ivy`
+- ✅ Works with both `ViewEngine` and `Ivy` (_note: if you want to use `ngx-template-streams` with **Ivy** you have to use the lastest version of `ngx-template-streams`_)
```
Comment (d3lm) [inline README.md]: Could you add an `*` after Ivy and add the note below the features?

Here's how I would change it:

```text
- ✅ Works with both `ViewEngine` and `Ivy` [\*](#ivy-compatibility-note)

...

**Notes**

[<a id="ivy-compatibility-note"></a>1] If you want to use `ngx-template-streams` with **Ivy** you have to use the **lastest** version of `ngx-template-streams`.
```

**Review by d3lm — CHANGES_REQUESTED** [review]
Thanks for the PR, really appreciate it! Left a comment on the readme note.

Also can you squash your commits into one and follow the commit convention? I suggest to go with something like:

`fix(lib): ivy compatibility`

Once that's done, I can merge it and publish a new release to npm.

**Comment by fxck** [comment]
Done, should we change dependencies as well though?

**Comment by d3lm** [comment]
Let's wait for that until Angular 9 is released. If it is I ll upgrade and cut a new release. Sounds good?

**Comment by fxck** [comment]
👍

**Comment by d3lm** [comment]
@fxck Can you give it a spin? I just published a new version.

**Comment by fxck** [comment]
Looks good at the first glance.

**Comment by d3lm** [comment]
Thanks for checking!

**Pull request merged**